python/lsst/obs/cfht/ingest.py

```diff
@@ -41,6 +46,34 @@
            }
```

Comment:
It doesn't look consistent with https://github.com/lsst/obs_cfht/pull/74/commits/d63ccc45bf2f455d1927627db5b32864c7b4a8c8 some filters are missing (the narrow band ones)

Reply:
Correct. That's the gen2 list. I was trying not to mess with the gen2 code paths.